lib/tests/modules.sh

```diff
@@ -439,6 +439,19 @@ checkConfigOutput '^"The option `a\.b. defined in `.*/doRename-warnings\.nix. ha
 checkConfigOutput '^"pear"$' config.once.raw ./merge-module-with-key.nix
 checkConfigOutput '^"pear\\npear"$' config.twice.raw ./merge-module-with-key.nix
 
+# Declaration positions
+# Line should be present for direct options
+checkConfigOutput '^[[:digit:]]+$' options.imported.nya.declarationsWithLocations.0.line ./declaration-positions.nix
```

Comment:
I'd prefer to have more literal matches in these tests, so that we can observe any changes if they happen, as they may represent regressions in the module system or in new Nix versions. Even if `unsafeGetAttrPos` isn't really part of Nix's contract, we'll want to know.

A test failure isn't the end of the world, and if we need more leniency, e.g. for a new Nix version we can add that later.

Reply:
changed